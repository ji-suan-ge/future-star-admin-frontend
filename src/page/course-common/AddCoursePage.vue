<template>
  <el-container>
    <el-main>
      <el-page-header @back="goBack()" content="新建课程"></el-page-header>
      <el-divider></el-divider>
      <el-card shadow="never">
        <el-form label-width="150px">
        <el-divider>课程信息</el-divider>
        <el-card shadow="never">
            <el-form-item label="课程名称：">
              <el-input
                v-model="course.name">
              </el-input>
            </el-form-item>
            <el-form-item label="课程起止时间：">
              <el-date-picker
                type="daterange"
                v-model="course.timeRange"
                range-separator="至"
                start-placeholder="开始时间"
                end-placeholder="结束时间"
                placeholder="选择时间范围">
              </el-date-picker>
            </el-form-item>
            <el-form-item label="课程地点：">
              <el-input
                v-model="course.location">
              </el-input>
            </el-form-item>
            <el-form-item label="课程介绍：">
              <el-input
                :autosize="{ minRows: 6, maxRows: 30}"
                v-model="course.introduction"
                type="textarea"/>
            </el-form-item>
        </el-card>
        <el-divider>讲师信息</el-divider>
        <el-card shadow="never">
          <el-row>
            <el-col :span="11">
              <el-form-item label="讲师名：">
                <el-input v-model="course.teacher.name"></el-input>
              </el-form-item>
            </el-col>
            <el-col :span="13">
              <el-form-item label="讲师联系方式：">
                <el-input v-model="course.teacher.contact_way"></el-input>
              </el-form-item>
            </el-col>
          </el-row>
          <el-form-item label="讲师头像：">
            <el-input v-model="course.teacher.avatar"></el-input>
          </el-form-item>
          <el-form-item label="讲师头衔：">
            <el-input v-model="course.teacher.title"></el-input>
          </el-form-item>
          <el-form-item label="讲师简介：">
            <el-input
              :autosize="{ minRows: 3, maxRows: 8}"
              v-model="course.teacher.introduction"
              type="textarea"/>
          </el-form-item>
        </el-card>
        <el-form-item>
          <el-row type="flex" justify="space-around">
            <el-col :span="3">
              <el-button type="primary" @click="createCourse" round>创建</el-button>
            </el-col>
            <el-col :span="3">
              <el-button @click="clearText" round>清空</el-button>
            </el-col>
          </el-row>
        </el-form-item>
      </el-form>
      </el-card>
    </el-main>
  </el-container>
</template>

<script>
export default {
  name: 'AddCoursePage',
  data: function () {
    return {
      currentClass: {},
      course: {
        name: '',
        introduction: '',
        location: '',
        timeRange: '',
        teacher: {
          name: '',
          avatar: '',
          title: '',
          introduction: '',
          contact_way: ''
        }
      }
    }
  },
  created () {
    this.currentClass = this.$store.getters.getCurrentClass
  },
  methods: {
    goBack: function () {
      this.$router.go(-1)
    },
    clearText: function () {
      this.course = {
        name: '',
        introduction: '',
        location: '',
        timeRange: '',
        teacher: {
          name: '',
          avatar: '',
          title: '',
          introduction: '',
          contact_way: ''
        }
      }
    },
    createCourse: async function () {
      let that = this
      let startTime = that.course.timeRange[0].toLocaleDateString().replace(/\//g, '-')
      let endTime = that.course.timeRange[1].toLocaleDateString().replace(/\//g, '-')
      await that.$store.dispatch('changeCurrentCourse', that.course)
      that.axios.post('/course/course', {
        clazz: that.currentClass.id,
        name: that.course.name,
        introduction: that.course.introduction,
        begin_time: startTime,
        end_time: endTime,
        location: that.course.location,
        teacher: {
          name: that.course.teacher.name,
          avatar: that.course.teacher.avatar,
          title: that.course.teacher.title,
          introduction: that.course.teacher.introduction,
          contact_way: that.course.teacher.contact_way
        }
      }).then(async function (response) {
        if (response.data.code === '2000') {
          await that.$router.push('/class-detail')
          that.$message({
            type: 'success',
            message: '创建成功',
            duration: 2000
          })
        } else {
          that.$message({
            type: 'error',
            message: '服务器繁忙，请稍后重试',
            duration: 2000
          })
        }
      }).catch(function (error) {
        console.log(error)
        that.$message({
          type: 'error',
          message: '服务器繁忙，请稍后重试',
          duration: 2000
        })
      })
    }
  }
}
</script>

<style scoped>

</style>
