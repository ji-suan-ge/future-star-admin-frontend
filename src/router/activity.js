import CreateActivityPage from '../page/CreateActivityPage'
import ShowActivityParticipantsPage from '../page/ShowActivityParticipantsPage'
import EditExistingActivityPage from '../page/EditExistingActivityPage'
import ShowActivityInformationPage from '../page/ShowActivityInformationPage'

export default [
  {
    path: '/create-activity',
    name: 'CreateActivityPage',
    component: CreateActivityPage
  },
  {
    path: '/show-participants',
    name: 'ShowActivityParticipantsPage',
    component: ShowActivityParticipantsPage
  },
  {
    path: '/edit-activity',
    name: 'EditExistingActivityPage',
    component: EditExistingActivityPage
  },
  {
    path: '/show-activity-details',
    name: 'ShowActivityInformationPage',
    component: ShowActivityInformationPage
  }
]
