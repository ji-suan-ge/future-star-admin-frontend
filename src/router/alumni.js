import ShowAlumniInformationPage from '../page/alumni-management/ShowAlumniInformationPage'
import EditAlumniInformationPage from '../page/alumni-management/EditAlumniInformationPage'
import ShowAlumniEditHistoryPage from '../page/alumni-management/ShowAlumniEditHistoryPage'
import ShowAlumniHistoryInformationPage from '../page/alumni-management/ShowAlumniHistoryInformationPage'

export default [
  {
    path: '/show-all-information',
    name: 'ShowAlumniInformationPage',
    component: ShowAlumniInformationPage
  },
  {
    path: '/edit-alumni-information',
    name: 'EditAlumniInformationPage',
    component: EditAlumniInformationPage
  },
  {
    path: '/show-edit-history',
    name: 'ShowAlumniEditHistoryPage',
    component: ShowAlumniEditHistoryPage
  },
  {
    path: '/show-history-information',
    name: 'ShowAlumniHistoryInformationPage',
    component: ShowAlumniHistoryInformationPage
  }
]
