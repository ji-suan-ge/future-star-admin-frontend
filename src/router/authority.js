import AddAdministratorPage from '../page/authority-management/AddAdministratorPage'

export default [
  {
    path: '/add-administrator',
    name: 'AddAdministratorPage',
    component: AddAdministratorPage
  }
]
