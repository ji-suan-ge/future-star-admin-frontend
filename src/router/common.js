import LoginPage from '../page/LoginPage'
import HomePage from '../page/HomePage'
import homeChildren from './home-children'

export default [
  {
    path: '/',
    name: 'Root',
    redirect: '/login'
  },
  {
    path: '/login',
    name: 'LoginPage',
    component: LoginPage
  },
  {
    path: '/home',
    name: 'HomePage',
    component: HomePage,
    redirect: '/home/welcome',
    children: [
      ...homeChildren
    ]
  }
]
