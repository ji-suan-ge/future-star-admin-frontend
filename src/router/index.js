import Vue from 'vue'
import VueRouter from 'vue-router'
import LoginPage from '../page/LoginPage'
import HomePage from '../page/HomePage'
import EnrollmentManagementPage from '../page/EnrollmentManagementPage'
import CourseManagementPage from '../page/CourseManagementPage'
import ActivityManagementPage from '../page/ActivityManagementPage'
import AlumniManagementPage from '../page/AlumniManagementPage'
import AuthorityManagementPage from '../page/AuthorityManagementPage'
import CreateActivityPage from '../page/CreateActivityPage'
import AddAdministrator from '../page/AddAdministrator'
import WelcomePage from '../page/WelcomPage'

Vue.use(VueRouter)

const routes = [
  {
    path: '/',
    name: 'Root',
    redirect: '/login'
  },
  {
    path: '/login',
    name: 'LoginPage',
    component: LoginPage
  },
  {
    path: '/home',
    name: 'HomePage',
    component: HomePage,
    redirect: '/home/welcome',
    children: [
      {
        path: 'welcome',
        name: 'WelcomePage',
        component: WelcomePage
      },
      {
        path: 'enrollment-management',
        name: 'EnrollmentManagementPage',
        component: EnrollmentManagementPage
      },
      {
        path: 'course-management',
        name: 'CourseManagementPage',
        component: CourseManagementPage
      },
      {
        path: 'activity-management',
        name: 'ActivityManagementPage',
        component: ActivityManagementPage
      },
      {
        path: 'alumni-management',
        name: 'AlumniManagementPage',
        component: AlumniManagementPage
      },
      {
        path: 'authority-management',
        name: 'AuthorityManagementPage',
        component: AuthorityManagementPage
      },
      {
        path: 'add-administrator',
        name: 'AddAdministrator',
        component: AddAdministrator
      }
    ]
  },
  {
    path: '/create-activity',
    name: 'CreateActivityPage',
    component: CreateActivityPage
  }
]

const router = new VueRouter({
  routes
})

export default router
