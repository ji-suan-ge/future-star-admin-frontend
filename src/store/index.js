import Vue from 'vue'
import Vuex from 'vuex'

Vue.use(Vuex)

export default new Vuex.Store({
  state: {
    activeIndexOfNavigation: '0',
    activeNameOfCourse: 'first',
    activeNameOfEnrollment: 'first',
    activeNameOfActivity: 'first',
    activeSemesterOfCourse: '',
    semesters: '',
    currentSemester: '',
    classes: '',
    currentClass: '',
    courses: '',
    currentCourse: '',
    infoOfAnAlumni: null
  },
  mutations: {
    changeActiveIndexOfNavigation (state, activeIndex) {
      state.activeIndexOfNavigation = activeIndex
    },
    changeActiveNameOfEnrollment (state, activeName) {
      state.activeNameOfEnrollment = activeName
    },
    changeActiveNameOfCourse (state, activeName) {
      state.activeNameOfCourse = activeName
    },
    changeActiveNameOfActivity (state, activeName) {
      state.activeNameOfActivity = activeName
    },
    changeActiveSemesterOfCourse (state, activeID) {
      state.activeSemesterOfCourse = activeID
    },
    changeSemesters (state, semesters) {
      state.semesters = semesters
    },
    changeInfoOfAnAlumni (state, alumniInfo) {
      state.infoOfAnAlumni = alumniInfo
    },
    changeCurrentSemester (state, semester) {
      state.currentSemester = semester
    },
    changeClasses (state, classes) {
      state.classes = classes
    },
    changeCurrentClass (state, currentClass) {
      state.currentClass = currentClass
    },
    changeCourses (state, courses) {
      state.courses = courses
    },
    changeCurrentCourse (state, currentCourse) {
      state.currentCourse = currentCourse
    },
    initializeStateWhileLogin (state) {
      state.activeIndexOfNavigation = '0'
      state.activeNameOfCourse = 'first'
      state.activeNameOfEnrollment = 'first'
      state.activeNameOfActivity = 'first'
      state.activeSemesterOfCourse = ''
      state.semesters = ''
      state.currentSemester = ''
      state.classes = ''
      state.currentClass = ''
      state.courses = ''
      state.currentCourse = ''
      state.infoOfAnAlumni = null
    },
    initializeStateWhileSwitch (state) {
      state.activeNameOfCourse = 'first'
      state.activeNameOfEnrollment = 'first'
      state.activeNameOfActivity = 'first'
      state.activeSemesterOfCourse = ''
      state.semesters = ''
      state.currentSemester = ''
      state.classes = ''
      state.currentClass = ''
      state.courses = ''
      state.currentCourse = ''
      state.infoOfAnAlumni = null
    }
  },
  actions: {
    changeActiveIndexOfNavigation (context, activeIndex) {
      context.commit('changeActiveIndexOfNavigation', activeIndex)
    },
    changeActiveNameOfEnrollment (context, activeName) {
      context.commit('changeActiveNameOfEnrollment', activeName)
    },
    changeActiveNameOfCourse (context, activeName) {
      context.commit('changeActiveNameOfCourse', activeName)
    },
    changeActiveNameOfActivity (context, activeName) {
      context.commit('changeActiveNameOfActivity', activeName)
    },
    changeActiveSemesterOfCourse (context, activeID) {
      context.commit('changeActiveSemesterOfCourse', activeID)
    },
    changeSemesters (context, semesters) {
      context.commit('changeSemesters', semesters)
    },
    changeInfoOfAnAlumni (context, alumniInfo) {
      context.commit('changeInfoOfAnAlumni', alumniInfo)
    },
    changeCurrentSemester (context, semester) {
      context.commit('changeCurrentSemester', semester)
    },
    changeClasses (context, classes) {
      context.commit('changeClasses', classes)
    },
    changeCurrentClass (context, currentClass) {
      context.commit('changeCurrentClass', currentClass)
    },
    changeCourses (context, courses) {
      context.commit('changeCourses', courses)
    },
    changeCurrentCourse (context, currentCourse) {
      context.commit('changeCurrentCourse', currentCourse)
    },
    initializeStateWhileLogin (context) {
      context.commit('initializeStateWhileLogin')
    },
    initializeStateWhileSwitch (context) {
      context.commit('initializeStateWhileSwitch')
    }
  },
  getters: {
    getActiveIndexOfNavigation (state) {
      return state.activeIndexOfNavigation
    },
    getActiveNameOfEnrollment (state) {
      return state.activeNameOfEnrollment
    },
    getActiveNameOfCourse (state) {
      return state.activeNameOfCourse
    },
    getActiveNameOfActivity (state) {
      return state.activeNameOfActivity
    },
    getActiveSemesterOfCourse (state) {
      return state.activeSemesterOfCourse
    },
    getSemesters (state) {
      return state.semesters
    },
    getInfoOfAnAlumni (state) {
      return state.infoOfAnAlumni
    },
    getCurrentSemester (state) {
      return state.currentSemester
    },
    getClasses (state) {
      return state.classes
    },
    getCurrentClass (state) {
      return state.currentClass
    },
    getCourses (state) {
      return state.courses
    },
    getCurrentCourse (state) {
      return state.currentCourse
    }
  },
  modules: {
  }
})
