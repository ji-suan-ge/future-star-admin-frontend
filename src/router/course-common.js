import DetailOfSemesterPage from '../page/course-common/DetailOfSemesterPage'
import DetailOfClassPage from '../page/course-common/DetailOfClassPage'
import DetailOfCoursePage from '../page/course-common/DetailOfCoursePage'
import DetailOfTeacherPage from '../page/course-common/DetailOfTeacherPage'
import DetailOfSubCoursePage from '../page/course-common/DetailOfSubCoursePage'
import EditSemesterInfoPage from '../page/course-common/EditSemesterInfoPage'
import EditClassInfoPage from '../page/course-common/EditClassInfoPage'
import EditCourseInfoPage from '../page/course-common/EditCourseInfoPage'
import EditTeacherInfoPage from '../page/course-common/EditTeacherInfoPage'
import AddCoursePage from '../page/course-common/AddCoursePage'
import AddCourseResourcePage from '../page/course-common/AddCourseResourcePage'
import StudentInClass from '../page/course-common/StudentInClass'

export default [
  {
    path: '/semester-detail',
    name: 'DetailOfSemesterPage',
    component: DetailOfSemesterPage
  },
  {
    path: '/class-detail',
    name: 'DetailOfClassPage',
    component: DetailOfClassPage
  },
  {
    path: '/course-detail',
    name: 'DetailOfCoursePage',
    component: DetailOfCoursePage
  },
  {
    path: '/teacher-detail',
    name: 'DetailOfTeacherPage',
    component: DetailOfTeacherPage
  },
  {
    path: '/sub-course-detail',
    name: 'DetailOfSubCoursePage',
    component: DetailOfSubCoursePage
  },
  {
    path: '/edit-semester-info',
    name: 'EditSemesterInfoPage',
    component: EditSemesterInfoPage
  },
  {
    path: '/edit-class-info',
    name: 'EditClassInfoPage',
    component: EditClassInfoPage
  },
  {
    path: '/edit-course-info',
    name: 'EditCourseInfoPage',
    component: EditCourseInfoPage
  },
  {
    path: '/edit-teacher-info',
    name: 'EditTeacherInfoPage',
    component: EditTeacherInfoPage
  },
  {
    path: '/add-course',
    name: 'AddCoursePage',
    component: AddCoursePage
  },
  {
    path: '/add-course-resource',
    name: 'AddCourseResourcePage',
    component: AddCourseResourcePage
  },
  {
    path: '/student-in-class',
    name: 'StudentInClass',
    component: StudentInClass
  }
]
