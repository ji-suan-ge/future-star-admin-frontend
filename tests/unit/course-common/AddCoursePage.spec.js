import { createLocalVue, mount } from '@vue/test-utils'
import AddCoursePage from '../../../src/page/course-common/AddCoursePage'
import Vuex from 'vuex'
import ElementUI from 'element-ui'
import VueRouter from 'vue-router'
import store from '../../../src/store'
import router from '../../../src/router'

describe('AddCoursePage', () => {
  const localVue = createLocalVue()
  localVue.use(Vuex)
  localVue.use(ElementUI)
  localVue.use(VueRouter)
  const wrapper = mount(AddCoursePage, {
    store,
    router,
    localVue
  })
  test('快照测试', async () => {
    await expect(wrapper.html()).toMatchSnapshot()
  })
  test('是一个 Vue 实例', async () => {
    await expect(wrapper.isVueInstance()).toBeTruthy()
  })
})
