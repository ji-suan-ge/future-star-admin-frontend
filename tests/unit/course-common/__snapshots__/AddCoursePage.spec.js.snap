// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddCoursePage 快照测试 1`] = `
<section class="el-container">
  <main class="el-main">
    <div class="el-page-header">
      <div class="el-page-header__left"><i class="el-icon-back"></i>
        <div class="el-page-header__title">返回</div>
      </div>
      <div class="el-page-header__content">新建课程</div>
    </div>
    <div class="el-divider el-divider--horizontal">
      <!---->
    </div>
    <div class="el-card is-never-shadow">
      <!---->
      <div class="el-card__body">
        <form class="el-form">
          <div class="el-divider el-divider--horizontal">
            <div class="el-divider__text is-center">课程信息</div>
          </div>
          <div class="el-card is-never-shadow">
            <!---->
            <div class="el-card__body">
              <div class="el-row">
                <div class="el-col el-col-12">
                  <div class="el-form-item"><label class="el-form-item__label" style="width: 150px;">课程名称：</label>
                    <div class="el-form-item__content" style="margin-left: 150px;">
                      <div class="el-input">
                        <!----><input type="text" autocomplete="off" class="el-input__inner">
                        <!---->
                        <!---->
                        <!---->
                        <!---->
                      </div>
                      <!---->
                    </div>
                  </div>
                </div>
                <div class="el-col el-col-8">
                  <div class="el-form-item"><label class="el-form-item__label" style="width: 150px;">类别：</label>
                    <div class="el-form-item__content" style="margin-left: 150px;">
                      <div class="el-input">
                        <!----><input type="text" autocomplete="off" class="el-input__inner">
                        <!---->
                        <!---->
                        <!---->
                        <!---->
                      </div>
                      <!---->
                    </div>
                  </div>
                </div>
              </div>
              <div class="el-form-item"><label class="el-form-item__label" style="width: 150px;">课程起止时间：</label>
                <div class="el-form-item__content" style="margin-left: 150px;">
                  <div class="el-date-editor el-range-editor el-input__inner el-date-editor--daterange"><i class="el-input__icon el-range__icon el-icon-date"></i><input autocomplete="off" placeholder="开始时间" name="" class="el-range-input"><span class="el-range-separator">至</span><input autocomplete="off" placeholder="结束时间" name="" class="el-range-input"><i class="el-input__icon el-range__close-icon"></i></div>
                  <!---->
                </div>
              </div>
              <div class="el-form-item"><label class="el-form-item__label" style="width: 150px;">课程地点：</label>
                <div class="el-form-item__content" style="margin-left: 150px;">
                  <div class="el-input">
                    <!----><input type="text" autocomplete="off" class="el-input__inner">
                    <!---->
                    <!---->
                    <!---->
                    <!---->
                  </div>
                  <!---->
                </div>
              </div>
              <div class="el-form-item"><label class="el-form-item__label" style="width: 150px;">课程介绍：</label>
                <div class="el-form-item__content" style="margin-left: 150px;">
                  <div class="el-textarea"><textarea autocomplete="off" class="el-textarea__inner" style="min-height: -24px; height: -120px;"></textarea>
                    <!---->
                  </div>
                  <!---->
                </div>
              </div>
            </div>
          </div>
          <div class="el-divider el-divider--horizontal">
            <div class="el-divider__text is-center">讲师信息</div>
          </div>
          <div class="el-card is-never-shadow">
            <!---->
            <div class="el-card__body">
              <div class="el-row">
                <div class="el-col el-col-11">
                  <div class="el-form-item"><label class="el-form-item__label" style="width: 150px;">讲师名：</label>
                    <div class="el-form-item__content" style="margin-left: 150px;">
                      <div class="el-input">
                        <!----><input type="text" autocomplete="off" class="el-input__inner">
                        <!---->
                        <!---->
                        <!---->
                        <!---->
                      </div>
                      <!---->
                    </div>
                  </div>
                </div>
                <div class="el-col el-col-13">
                  <div class="el-form-item"><label class="el-form-item__label" style="width: 150px;">讲师联系方式：</label>
                    <div class="el-form-item__content" style="margin-left: 150px;">
                      <div class="el-input">
                        <!----><input type="text" autocomplete="off" class="el-input__inner">
                        <!---->
                        <!---->
                        <!---->
                        <!---->
                      </div>
                      <!---->
                    </div>
                  </div>
                </div>
              </div>
              <div class="el-form-item"><label class="el-form-item__label" style="width: 150px;">讲师头衔：</label>
                <div class="el-form-item__content" style="margin-left: 150px;">
                  <div class="el-input">
                    <!----><input type="text" autocomplete="off" class="el-input__inner">
                    <!---->
                    <!---->
                    <!---->
                    <!---->
                  </div>
                  <!---->
                </div>
              </div>
              <div class="el-form-item"><label class="el-form-item__label" style="width: 150px;">讲师简介：</label>
                <div class="el-form-item__content" style="margin-left: 150px;">
                  <div class="el-textarea"><textarea autocomplete="off" class="el-textarea__inner" style="min-height: -12px; height: -32px;"></textarea>
                    <!---->
                  </div>
                  <!---->
                </div>
              </div>
            </div>
          </div>
          <div class="el-form-item">
            <!---->
            <div class="el-form-item__content" style="margin-left: 150px;">
              <div class="el-row is-justify-space-around el-row--flex">
                <div class="el-col el-col-3"><button type="button" class="el-button el-button--primary is-round">
                    <!---->
                    <!----><span>创建</span></button></div>
                <div class="el-col el-col-3"><button type="button" class="el-button el-button--default is-round">
                    <!---->
                    <!----><span>清空</span></button></div>
              </div>
              <!---->
            </div>
          </div>
        </form>
      </div>
    </div>
  </main>
</section>
`;
